src/Data/MemoTrie.hs

```diff
@@ -445,6 +446,7 @@ WordInstance(Word8,Word8Trie)
 WordInstance(Word16,Word16Trie)
 WordInstance(Word32,Word32Trie)
 WordInstance(Word64,Word64Trie)
+WordInstance(Natural,NaturalTrie)
```

Comment:
Thanks for the PR! Does this line need `#if`-protecting as with the corresponding import (line 69)?